src/org/usfirst/frc2876/PowerUp2018/commands/AutoCGSwitchScale.java

```diff
@@ -0,0 +1,95 @@
+package org.usfirst.frc2876.PowerUp2018.commands;
+
+import org.usfirst.frc2876.PowerUp2018.Robot;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+//  DECISION TABLE:
+//    Note: Assumption is that the team that argued for center will attempt cube in switch
+//
+//                     Game Data
+//    RobotPos     LL*      LR*      RL*      RR*
+//    Center       <----------do switch -------->
+//    Left         scale    switch   scale    scale
+//    Right        scale    scale    switch   scale
+
+/**
+ *
+ */
+public class AutoCGSwitchScale extends CommandGroup {
+
+	// Guesstimates!
+	private static final int CENTER_WALL_TO_TURN_DIST = 20;
+	private static final int WALL_TO_SWITCH_DIST = 50;
+	private static final int AT_SWITCH_DIST = 10;
+	private static final int WALL_TO_SCALE_DIST = 100;
+	private static final int WALL_TO_PLATFORM_ZONE_DIST = 75;
+	private static final int PLATFORM_ZONE_WIDTH_DIST = 75;
+	private static final int PLATFORM_ZONE_TO_SCALE_DIST = 15;
+	private static final int AT_SCALE_DIST = 5;
+	
+	
+    public AutoCGSwitchScale() {
+    	
+    	System.out.println("Running AutoCGSwitchScale: " + Robot.getRobotPos() + " " + Robot.isSwitchLeft());
+		
+		// TODO we need to raise arms to release kick-stand
+		// addParallel(new ElevatorPosition(10));
```

Comment:
If going for scale, don't raise elevator right away to keep center of gravity low.  We should test and see what happens driving around with cube/intake in up position.  Maybe it will be less likely to fall out?